### PR TITLE
Fixed applicant field in forms

### PR DESCRIPTION
**Pull request by AdarshNandanwar**
The homepage for students is still student/dashboard and not student/home. /student/home is not in urls.py. Thus I have not changed form redirect on submission to /home.

### CODE REVIEW

File: transfers/models.py [hunk 1]
```diff
@@ -27,7 +27,7 @@ class UserProfile(models.Model):
     ]
 
     user = models.OneToOneField(User, primary_key=True,
-        on_delete=models.CASCADE)
+        on_delete=models.CASCADE, related_name="userprofile")
```
Comment (ChetanKhanna) [inline transfers/models.py]: Why is this needed?

**Review by ChetanKhanna — CHANGES_REQUESTED** [review]
There is no students/home. You need to redirect to students/dashboard

File: transfers/models.py (hunk 1 above)
Comment (AdarshNandanwar) [reply transfers/models.py]: To get UserProfile from request.user which is needed to get who is logged in currently.

**Comment by ChetanKhanna** [comment]
You can simply use `user.userprofile`

File: transfers/models.py (hunk 1 above)
Comment (ChetanKhanna) [reply transfers/models.py]: You can simply user `user.userprofile`

**Review by ChetanKhanna — APPROVED** [review]
